node/repo/boostrepo.go

```diff
@@ -0,0 +1,31 @@
+package repo
```

Comment:
It's not a problem but I'm curious why move the Boost repo implementation from where it was?

Reply:
Initially I was keeping backupmgr within modules, that was creating a cyclic import. To avoid this I moved it to it's own package. Although, I moved backup out of module to its own package, but this cyclic dependency issue might come again for someone working on module section. Keeping that in mind, I did not move it back.